src/main/java/org/jenkinsci/test/acceptance/plugins/matrix_auth/MatrixAuthorizationStrategy.java

```diff
@@ -20,8 +21,13 @@ public MatrixAuthorizationStrategy(GlobalSecurityConfig context, String path) {
      * Adds a new user/group to this matrix.
      */
     public MatrixRow addUser(String name) {
-        runThenHandleAlert(() -> this.name.resolve().findElement(by.parent()).findElement(by.button("Add user or group…")).click(),
-                a -> {
+        runThenHandleAlert(() -> {
+            WebElement button = this.name.resolve().findElement(by.parent())
+                                         .findElement(by.button("Add user or group…"));
+            // JENKINS-63159 if the user has just (un)checked a permission then the button click may fail as the button
+            // is obscured by a tooltip
+            button.sendKeys(" "); // because this is reliable and button.click() is not due to the tooltip.
```

Comment:
I admit this is hacky, nut I could not find a way to make sure there was no tooltip or the button was otherwise not obscured.